### PR TITLE
Define common sentences as constants

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
*Define common sentences as constants，You don't have to edit it multiple times.*

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/addons/dns/dns.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ const (
 	kubeDNSStubDomain          = "stubDomains"
 	kubeDNSUpstreamNameservers = "upstreamNameservers"
 	kubeDNSFederation          = "federations"
+	UnableDecodeCoreDNS        = "unable to decode CoreDNS "
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/dns.go]: please rename to UnableToDecodeCoreDNS

File: cmd/kubeadm/app/phases/addons/dns/dns.go [hunk 2]
```diff
@@ -221,7 +222,7 @@ func coreDNSAddon(cfg *kubeadmapi.ClusterConfiguration, client clientset.Interfa
 func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, client clientset.Interface) error {
 	coreDNSConfigMap := &v1.ConfigMap{}
 	if err := kuberuntime.DecodeInto(clientsetscheme.Codecs.UniversalDecoder(), configBytes, coreDNSConfigMap); err != nil {
-		return errors.Wrap(err, "unable to decode CoreDNS configmap")
+		return errors.Wrap(err, UnabledeCodeCoreDNS+"configmap")
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/dns.go]: there is a typo in the constant name, but also please use:
`errors.Wrapf(err, "%s ConfigMap", UnableToDecodeCoreDNS)`

please uppercase all of these:
configmap -> ConfigMaps
clusterroles -> ClusterRole
clusterolebindings -> ClusterRoleBinding
serviceaccount -> ServiceAccount
deployment -> Deployment

**Review by neolit123 — COMMENTED** [review]
thanks @xichengliudui 
i've added a couple of comments.

/kind cleanup
/priority backlog
/approve

**Comment by xichengliudui** [comment]
Thank you very much. I have modified it.

File: cmd/kubeadm/app/phases/addons/dns/dns.go [hunk 3]
```diff
@@ -221,7 +222,7 @@ func coreDNSAddon(cfg *kubeadmapi.ClusterConfiguration, client clientset.Interfa
 func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, client clientset.Interface) error {
 	coreDNSConfigMap := &v1.ConfigMap{}
 	if err := kuberuntime.DecodeInto(clientsetscheme.Codecs.UniversalDecoder(), configBytes, coreDNSConfigMap); err != nil {
-		return errors.Wrap(err, "unable to decode CoreDNS configmap")
+		return errors.Wrap(err, "%s ConfigMaps", UnableToDecodeCoreDNS)
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/dns.go]: should use `Wrapf` also `ConfigMaps` -> `ConfigMap`

File: cmd/kubeadm/app/phases/addons/dns/dns.go [hunk 4]
```diff
@@ -231,7 +232,7 @@ func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, clien
 
 	coreDNSClusterRoles := &rbac.ClusterRole{}
 	if err := kuberuntime.DecodeInto(clientsetscheme.Codecs.UniversalDecoder(), []byte(CoreDNSClusterRole), coreDNSClusterRoles); err != nil {
-		return errors.Wrap(err, "unable to decode CoreDNS clusterroles")
+		return errors.Wrap(err, "%s ClusterRole", UnableToDecodeCoreDNS)
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/dns.go]: should be `Wrapf` because we are passing an argument.

Comment (xichengliudui) [reply cmd/kubeadm/app/phases/addons/dns/dns.go]: Thank you, it has been modified.

**Comment by neolit123** [comment]
thanks
/lgtm

File: cmd/kubeadm/app/phases/addons/dns/dns.go [hunk 5]
```diff
@@ -45,6 +45,7 @@ const (
 	kubeDNSStubDomain          = "stubDomains"
 	kubeDNSUpstreamNameservers = "upstreamNameservers"
 	kubeDNSFederation          = "federations"
+	UnableToDecodeCoreDNS      = "unable to decode CoreDNS"
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/addons/dns/dns.go]:
```
Errors from golint:
cmd/kubeadm/app/phases/addons/dns/dns.go:48:2: exported const UnableToDecodeCoreDNS should have comment (or a comment on this block) or be unexported
```

please lowercase the variable as it's used only in this file.

**Comment by xichengliudui** [comment]
@neolit123  I think there should be no problem this time. Thank you very much for your patient reply.

**Pull request merged**